chiron/neighbors.py

```diff
@@ -39,21 +61,10 @@ def wrap(self, xyz: jnp.array) -> jnp.array:
 
 class OrthogonalPeriodicSpace(Space):
     """
-    Calculate the periodic distance between two points.
+    Defines the simulation space for an orthogonal periodic system.
 
-    Returns
-    -------
-    Callable
```

Comment:
Is the return type correct here?

Reply:
I had initially set this up as nested functions.  I removed this when I updated the code in the PR.